containers/docker-compose.yaml

```diff
@@ -98,6 +98,11 @@ services:
       - QUARTERDECK_LOG_LEVEL=info
       - QUARTERDECK_CONSOLE_LOG=true
       - QUARTERDECK_ALLOW_ORIGINS=http://localhost:3000
+      - QUARTERDECK_VERIFY_BASE_URL=http://localhost:8080/verify
+      - QUARTERDECK_SENDGRID_API_KEY
+      - QUARTERDECK_SENDGRID_FROM_EMAIL=quarterdeck@rotational.io
+      - QUARTERDECK_SENDGRID_ADMIN_EMAIL=admins@rotational.io
```

Comment:
We should create a story to setup this Google Group so that we can receive emails at this email address (similar to support@rotational.io). I'd suggest `admin@rotational.io` -- @rebeccabilbro any thoughts on this?